source/projects/private-voice-assistant/voice_remote_classic_phone.markdown

```diff
@@ -0,0 +1,154 @@
+1. Add the **Voice over IP** integration.   
+    ![Voice over IP integration](/images/assist/voip_install.png)
+1. Once you see the integration, pick up the phone.
+   * You should now hear the message *This is your smart home speaking. Your phone is connected, but you must configure it within Home Assistant.*
+   * The integration should now include a device and entities.
+    ![Voice over IP integration with device and entities](/images/assist/voip_device_available.png)
+   * Don't hear the voice? Try these [troubleshooting steps](projects/private-voice-assistant/voice_remote_classic_phone/#troubleshoot-grandstream).
+1. Allow calls.
+   * Calls from new devices are blocked by default since voice commands could be used to control sensitive devices, such as locks and garage doors.
+   * In the **Voice over IP** integration, select the **device** link.
+   * To allow this phone to control your smart home, under **Configuration**, enable **Allow calls**.   
+   ![Voice over IP integration - allow calls](/images/assist/voip_configuration.png) 
+1. Your phone is set up in Home Assistant. The next step is to set up a voice assistant.
+
+## Setting up a Home Assistant voice assistant
+
+1. Set up text-to-speech in Home Assistant Cloud.
+   * Once the pipeline is further developed, the cloud won't be necessary. It will be possible to process speech locally. 
+   * Go to {% my cloud title="**Settings** > **Home Assistant Cloud**" %}.
+   * If you have multiple Home Assistance instances running, make sure you are not logged in to Home Assistance Cloud on another instance. 
+   * On your current instance, log in to your Home Assistant Cloud.
+   * Under **Text-to-speech**, select the default language and select whether you want the voice to sound more female or male.
+   ![Home Assistant cloud language settings](/images/assist/ha-cloud-tts-01.png) 
+1. Go to **Settings** > **Voice assistants** and select **Add assistant**.
+   * Enter a name. You can pick any name that is meaningful to you.
+   ![Enter a name for your voice assistant](/images/assist/assistant-give-name.png)
+   * Select a **Conversation agent**. This is the software that processes your voice stream. 
+   For this tutorial, select **Home Assistant**.
+     * Select the **Language** that you want to speak.
+      ![Select the language you want to speak](/images/assist/assistant-conversation-agent.png)
+   * In the **Speech-to-text** and **Text-to-speech** drop-down menus, select an assistant. For this tutorial, select **Home Assistant Cloud**.
+   * Under **Text-to-speech**, select the **voice** of the assistant. For this tutorial, use the default voice.
+   * Select **Save**.
+   ![Enter a name for your voice assistant](/images/assist/assistant-stt-tts-define.png)
+1. To be able to control your devices over a voice command, you must expose your entities to Assist:
```

Comment:
This should probably be moved to a specific documentation page about exposing entities. As it is relevant for any Assist Pipeline, but also Google/Alexa.